Source/Core/VideoBackends/Vulkan/VKTexture.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <memory>
+#include <vulkan/vulkan.h>
+
+#include "VideoCommon/AbstractTexture.h"
+#include "VideoCommon/BPStructs.h"
+#include "VideoCommon/VideoCommon.h"
+
+namespace Vulkan
+{
+
+class Texture2D;
+
+class VKTexture : public AbstractTexture
+{
+public:
+
+  Texture2D* GetRawTexIdentifier() const { return m_texture.get(); }
```

Comment:
These two functions should have their implementations in the cpp file. They should also be placed after the virtual functions.